src/public/constants.ts

```diff
@@ -14,6 +14,7 @@ export const enum FrameContexts {
   remove = 'remove',
   task = 'task',
   sidePanel = 'sidePanel',
+  stageView = 'stageView',
```

Comment:
NIT: should we just make it 'stage'?

Reply:
If we change the const then we will need to make a corresponding change in the consumer. Not a problem, just keep us updated.